contracts/season3/TalentSponsorship.sol

```diff
@@ -50,14 +54,20 @@ contract TalentSponsorship {
         string symbol
     );
 
-    constructor() {
+    constructor(address contractOwner) {
         totalSponsorships = 0;
         totalRevokedSponsorships = 0;
         totalClaimedSponsorships = 0;
+        transferOwnership(contractOwner);
```

Comment:
I think it's simpler if we just use the msg.sender instead of passing a custom address. This way we don't have any dependences while deploying the contract. What do you think?

Reply:
Since I don't know yet if the owner will be the wallet deploying the contract I did it this way. @francisco-leal we will deploy with the owner wallet?